sdk/eventhubs/azure-messaging-eventhubs/CHANGELOG.md

```diff
@@ -1,14 +1,6 @@
 # Release History
 
-## 1.0.0-beta.3 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+## 1.0.0-beta.3 (2023-10-10)
 
 ## 1.0.0-beta.2 (2023-09-12)
```

Comment:
@LarryOsterman, we need a changelog entry here.

Reply:
Added.

Reply:
Thank you!